cp3pt0-deployment/common/migrate_singleton.sh

```diff
@@ -90,21 +90,33 @@ function main() {
 function migrate_license_service_reporter(){
     title "LSR migration from ibm-cmmon-services to ${LSR_NAMESPACE}"
 
-    local lsr_instances=$("$OC" get IBMLicenseServiceReporter instance -n ${OPERATOR_NS} --no-headers | wc -l)
-    if [[ lsr_instances -eq 0 ]]; then
+    local count=$("${OC}" get IBMLicenseServiceReporter -n ${OPERATOR_NS} --no-headers | wc -l)
+
+    if [[ count -eq 0 ]]; then
         info "No LSR for migration found in ${OPERATOR_NS} namespace"
         return 0
     fi
 
-    lsr_pv_nr=$("${OC}" get pv -l license-service-reporter-pv=true --no-headers | wc -l )
-    if [[ lsr_pv_nr -ne 1 ]]; then
-        warning "Expecting exactly one PV with label license-service-reporter-pv=true. $lsr_pv_nr found. Migration skipped."
-        return 0
-    fi
+    lsr_cr_name_list=$("${OC}" get IBMLicenseServiceReporter -n ${OPERATOR_NS} --no-headers | awk '{print $1}')
```

Comment:
If there will be more than 1 instance for LSR, there should be msg/warning that expecting only one LSR instance.